geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/ExportCommand.scala

```diff
@@ -280,6 +280,11 @@ object ExportCommand extends LazyLogging {
       }
       val order = if (params.sortDescending) { SortOrder.DESCENDING } else { SortOrder.ASCENDING }
       query.setSortBy(fields.map(f => org.locationtech.geomesa.filter.ff.sort(f, order)).toArray)
+    } else if (hints.isArrowQuery) {
```

Comment:
This makes sense.  What did we do before?  

How does one call for an Arrow sort from the command line?

Reply:
this is just to ensure that any arrow sort hints get set as a regular query sort. in geomesa it doesn't matter if it's in one or both places, we will handle it the same. but for other data sources (e.g. postgis or whatever), we want to make sure that the results come back already sorted, so that we don't need complicated sort logic in this ArrowExporter.

Reply:
on the command line you can either use `--sort-by` or `--hints ARROW_SORT_FIELD=foo`